tests/Support/Settings.php

```diff
@@ -121,6 +121,10 @@ public function enableBadPasswordLdap(): Settings
             'ldap_basedn' => 'CN=Users,DC=ad,DC=example,Dc=com'
         ]);
     }
+    public function setEula($text = 'Default EULA text')
+    {
+        return $this->update(['default_eula_text' => $text]);
```

Comment:
What is this doing?

Reply:
Yeah it's only used for testing. In this case it's in `NotificationTest@testDefaultEulaIsSentWhenSetInCategory` on line 40. It's setting it in the application's global settings so we can assert against it further down.